lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
Ah, if you run it second time? In Portal this won't happen, since we always regenerate these files, but on theory someone may really exec this on the same file more than once.
OK than, but I will leave @jbalsas to take a look too, he has a fresh look on Portal and he may spot something else. Chema, if you are OK with these changes too, you may merge it.

Thanks,